src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexRunner.cs

```diff
@@ -699,7 +709,10 @@ string DescribeTextPosition()
 
                 if (runtextpos > runtextbeg)
                 {
-                    sb.Append(RegexCharClass.DescribeChar(runtext![runtextpos - 1]));
+                    if (runtext != null)
+                    {
+                        sb.Append(RegexCharClass.DescribeChar(runtext[runtextpos - 1]));
+                    }
```

Comment:
We should consider whether any of this tracing is still valuable, especially if dumping the current state won't help with the newer modes of access, and whether it should all be deleted or simplified significantly.  Entirely up to you, as likely the sole person who would ever enable it :)

Reply:
sure, I can handle that in a separate PR if that's ok.

Reply:
Yup